src/System.DirectoryServices/src/System/DirectoryServices/SearchWaitHandler.cs

```diff
@@ -15,34 +15,34 @@ namespace System.DirectoryServices
     using System.Reflection;
     using System.Security.Permissions;
     using System.Globalization;
+    // https://support.microsoft.com/en-us/kb/833789
```

Comment:
This code was added at some point to work around an OS issue.  Before I just remove it I want to make sure it's not needed anymore.